src/acq/adc.c

```diff
@@ -199,6 +199,7 @@ static void bl_acq_adc_group_disable(bl_acq_adc_group_t *adc_group)
 typedef struct
```

Comment:
Typo in commit message, please change to "acquire".

Reply:
+1